src/Microsoft.CodeAnalysis.FxCopAnalyzers/RulesMissingDocumentation.md

```diff
@@ -2,5 +2,6 @@
 
 Rule ID | Missing Help Link | Title |
 --------|-------------------|-------|
-CA1416 | https://docs.microsoft.com/visualstudio/code-quality/ca1416 | Validate platform compatibility |
+CA2218 | https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2218 | Override GetHashCode on overriding Equals |
+CA2224 | https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2224 | Override Equals on overloading operator equals |
```

Comment:
@gewarren seems like we missed porting docs for `CA2218` and `CA2224` to dotnet docs repo. The pages still exist in VS docs repo, though these have been ported as analyzers. Likely I missed them in the initial work.

Reply:
Ah, seems like I missed `CA2218` as it was converted to a VB only analyzer: https://github.com/dotnet/roslyn-analyzers/blob/f15404b312295e7cee16fb40b4c8d3f91f10f087/src/NetAnalyzers/VisualBasic/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/BasicOverrideGetHashCodeOnOverridingEquals.vb#L11-L16

C# does not need it.

Reply:
Same for `CA2224`: https://github.com/dotnet/roslyn-analyzers/blob/f15404b312295e7cee16fb40b4c8d3f91f10f087/src/NetAnalyzers/VisualBasic/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/BasicOverrideEqualsOnOverloadingOperatorEquals.vb#L11-L16

Reply:
Filed https://github.com/dotnet/docs/issues/20917 for doc porting.